script.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function () {
```

Comment:
A simple script for a simple app. Very succinct. Well named variables. One idea per line of code.